### PR TITLE
Fix For 'Entity Enrichment - Generic v2' Playbook

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8964

## Description
As part of the generic sub-playbook update effort, I reviewed 'Entity Enrichment - Generic v2' and noticed the following issues:
1- Missing playbook input - The 'UseReputationCommand' playbook input has been added to define whether reputation command should be used during enrichment processes. The input is mandatory and has a default value of 'false'. The playbook input.
2- Sub-playbook input configurations - 'UseReputationCommand' sub-playbook input configurations have been changed to use 'UseReputationCommand' playbook input configurations.
3- Missing playbook outputs.

## Screenshot
Entity_Enrichment_-_Generic_v2

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Entity_Enrichment_-_Generic_v2.yml [hunk 1]
```diff
@@ -57,23 +62,26 @@ tasks:
...
   "16":
     id: "16"
-    taskid: afe0c443-753d-4633-89e1-0d77dc533447
+    taskid: 79119615-b605-486c-8592-d84ab35523d3
     type: playbook
     task:
-      id: afe0c443-753d-4633-89e1-0d77dc533447
+      id: 79119615-b605-486c-8592-d84ab35523d3
       version: -1
       name: IP Enrichment - Generic v2
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Entity_Enrichment_-_Generic_v2.yml:77]: need to add to the original sub-playbook that we are using the following command there
`* IP address Reputation using !ip`

File: Packs/CommonPlaybooks/Playbooks/playbook-Entity_Enrichment_-_Generic_v2.yml [hunk 2]
```diff
@@ -110,15 +122,23 @@ tasks:
...
     type: playbook
     task:
-      id: 8189698a-475c-433b-8fbf-20275b3b2084
+      id: 6b49c26d-0f94-4a7d-82f3-cb91ec5d4530
       version: -1
       name: File Enrichment - Generic v2
-      description: ""
+      description: |-
+        Enrich a file using one or more integrations.
+
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Entity_Enrichment_-_Generic_v2.yml:140]: need to add to the original sub-playbook that we are using the following command there
`* File Reputation using !file`

**Review by ssokolovich — COMMENTED** [review]
just don't forget the RN :)

File: Packs/CommonPlaybooks/Playbooks/playbook-Entity_Enrichment_-_Generic_v2.yml (hunk 1 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Entity_Enrichment_-_Generic_v2.yml:77]: done

File: Packs/CommonPlaybooks/Playbooks/playbook-Entity_Enrichment_-_Generic_v2.yml (hunk 2 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Entity_Enrichment_-_Generic_v2.yml:140]: done

**Pull request closed without merging**